### ISSUE TITLE
Add Until.elementTextDoesNotMatch

### ISSUE THREAD

**Issue by DavidDeSloovere** [flood-io/element#37]
**Is your feature request related to a problem? Please describe.**
I want to check for an element to not match a specific text. That way I know it has changed after ie. button click. Writing an inverted regex pattern is not easy.

**Describe the solution you'd like**
Add Until.~DoesNotMatch as counterparts for Until.~Matches
- elementTextMatches(selectorOrLocator, regex) >> elementTextDoesNotMatch(selectorOrLocator, regex)
- titleMatches(title) >> titleDoesNotMatch(title)
- urlMatches(title) >> urlDoesNotMatch(title)

**Describe alternatives you've considered**
Inverted regex is something you'll need to look up, exposing a friendly alternative is nicer.

**Additional context**
I was able to figure out how to check for change with the following:
```
const regex = new RegExp("^(?!" + amountBefore + "$)");
await browser.wait(Until.elementTextMatches(amountLocator, regex));
```

Adding ~DoesNotMatch would allow this much simpler code:
```
const regex = new RegExp(amountBefore);
await browser.wait(Until.elementTextDoesNotMatch(amountLocator, regex));
```

**Comment by ivanvanderbyl** [comment]
@DavidDeSloovere thanks for the suggestion. I think this fits the API surface nicely. We'd happily accept a pull request with these changes + tests, and we'll include it in v1.1

**Comment by DavidDeSloovere** [comment]
I'd be happy to contribute. I should be able to make some time for this during hacktober.

**Comment by huyennbl** [comment]
Interested in this ticket, I'm taking this if nobody is working on it.

**Comment by DavidDeSloovere** [comment]
Good for me!

**Comment by ivanvanderbyl** [comment]
Implemented in #139
